nilearn/plotting/html_surface.py

```diff
@@ -596,9 +570,11 @@ def view_surf(
     if surf_map is None:
         surf_map = np.ones(len(surf_mesh[0]))
     else:
-        surf_mesh, surf_map = surface.check_mesh_and_data(surf_mesh, surf_map)
+        surf_mesh, surf_map = surface.surface.check_mesh_and_data(
+            surf_mesh, surf_map
+        )
     if bg_map is not None:
-        _, bg_map = surface.check_mesh_and_data(surf_mesh, bg_map)
+        _, bg_map = surface.surface.check_mesh_and_data(surf_mesh, bg_map)
```

Comment:
maybe for another PR but maybe we should just import the functions we need and not the whole module to avoid doing this kind `surface.surface.function_with_an_explicit_name_but_oh_no_now_are_beyond_the_allowed_line_length`

Reply:
🤣